### ISSUE TITLE
2025-04-17, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#143]
# Today's Arxiv Papers

(1) [BitNet b1.58 2B4T Technical Report](http://arxiv.org/abs/2504.12285v1), 2025-04-16

(2) [HLS-Eval: A Benchmark and Framework for Evaluating LLMs on High-Level Synthesis Design Tasks](http://arxiv.org/abs/2504.12268v1), 2025-04-16

(3) [FLIP Reasoning Challenge](http://arxiv.org/abs/2504.12256v1), 2025-04-16

(4) [AnomalyGen: An Automated Semantic Log Sequence Generation Framework with LLM for Anomaly Detection](http://arxiv.org/abs/2504.12250v1), 2025-04-16

(5) [MOS: Towards Effective Smart Contract Vulnerability Detection through Mixture-of-Experts Tuning of Large Language Models](http://arxiv.org/abs/2504.12234v1), 2025-04-16

(6) [Watermarking Needs Input Repetition Masking](http://arxiv.org/abs/2504.12229v1), 2025-04-16

(7) [d1: Scaling Reasoning in Diffusion Large Language Models via Reinforcement Learning](http://arxiv.org/abs/2504.12216v1), 2025-04-16

(8) [From Requirements to Architecture: Semi-Automatically Generating Software Architectures](http://arxiv.org/abs/2504.12192v1), 2025-04-16

(9) [What Do Large Language Models Know? Tacit Knowledge as a Potential Causal-Explanatory Structure](http://dx.doi.org/10.1017/psa.2025.19), 2025-04-16

(10) [SALAD: Improving Robustness and Generalization through Contrastive Learning with Structure-Aware and LLM-Driven Augmented Data](http://arxiv.org/abs/2504.12185v1), 2025-04-16

(11) [Trusting CHATGPT: how minor tweaks in the prompts lead to major differences in sentiment classification](http://arxiv.org/abs/2504.12180v1), 2025-04-16

(12) [ARCeR: an Agentic RAG for the Automated Definition of Cyber Ranges](http://arxiv.org/abs/2504.12143v1), 2025-04-16

(13) [Multilingual Contextualization of Large Language Models for Document-Level Machine Translation](http://arxiv.org/abs/2504.12140v1), 2025-04-16

(14) [Clarifying Ambiguities: on the Role of Ambiguity Types in Prompting Methods for Clarification Generation](http://arxiv.org/abs/2504.12113v1), 2025-04-16

(15) [Towards LLM Agents for Earth Observation](http://arxiv.org/abs/2504.12110v1), 2025-04-16

(16) [Entropy-Guided Watermarking for LLMs: A Test-Time Framework for Robust and Traceable Text Generation](http://arxiv.org/abs/2504.12108v1), 2025-04-16

(17) [Gauging Overprecision in LLMs: An Empirical Study](http://arxiv.org/abs/2504.12098v1), 2025-04-16

(18) [Reasoning-Based AI for Startup Evaluation (R.A.I.S.E.): A Memory-Augmented, Multi-Step Decision Framework](http://arxiv.org/abs/2504.12090v1), 2025-04-16

(19) [Selective Demonstration Retrieval for Improved Implicit Hate Speech Detection](http://arxiv.org/abs/2504.12082v1), 2025-04-16

(20) [Subitizing-Inspired_Large_Language_Models_for_Floorplanning](http://arxiv.org/abs/2504.12076v1), 2025-04-16

(21) [Optimizing Compound Retrieval Systems](http://dx.doi.org/10.1145/3726302.3730051), 2025-04-16

(22) [Modular-Cam: Modular Dynamic Camera-view Video Generation with LLM](http://arxiv.org/abs/2504.12048v1), 2025-04-16

(23) [OpDiffer: LLM-Assisted Opcode-Level Differential Testing of Ethereum Virtual Machine](http://dx.doi.org/10.1145/3728946), 2025-04-16

(24) [Instruction-augmented Multimodal Alignment for Image-Text and Element Matching](http://arxiv.org/abs/2504.12018v1), 2025-04-16

(25) [Active Human Feedback Collection via Neural Contextual Dueling Bandits](http://arxiv.org/abs/2504.12016v1), 2025-04-16

(26) [Purposefully Induced Psychosis (PIP): Embracing Hallucination as Imagination in Large Language Models](http://arxiv.org/abs/2504.12012v1), 2025-04-16

(27) [Generative Recommendation with Continuous-Token Diffusion](http://arxiv.org/abs/2504.12007v1), 2025-04-16

(28) [Language Models as Quasi-Crystalline Thought: Structure, Constraint, and Emergence in Generative Systems](http://arxiv.org/abs/2504.11986v1), 2025-04-16

(29) [SemEval-2025 Task 3: Mu-SHROOM, the Multilingual Shared Task on Hallucinations and Related Observable Overgeneration Mistakes](http://arxiv.org/abs/2504.11975v1), 2025-04-16

(30) [LLM-as-a-Judge: Reassessing the Performance of LLMs in Extractive QA](http://arxiv.org/abs/2504.11972v1), 2025-04-16

(31) [Robust and Fine-Grained Detection of AI Generated Texts](http://arxiv.org/abs/2504.11952v1), 2025-04-16

(32) [Mind2Matter: Creating 3D Models from EEG Signals](http://arxiv.org/abs/2504.11936v1), 2025-04-16

(33) [An LLM-as-a-judge Approach for Scalable Gender-Neutral Translation Evaluation](http://arxiv.org/abs/2504.11934v1), 2025-04-16

(34) [Rethinking the Generation of High-Quality CoT Data from the Perspective of LLM-Adaptive Question Difficulty Grading](http://arxiv.org/abs/2504.11919v1), 2025-04-16

(35) [AnomalyR1: A GRPO-based End-to-end MLLM for Industrial Anomaly Detection](http://arxiv.org/abs/2504.11914v1), 2025-04-16

(36) [Finding Flawed Fictions: Evaluating Complex Reasoning in Language Models via Plot Hole Detection](http://arxiv.org/abs/2504.11900v1), 2025-04-16

(37) [Rethinking LLM-Based Recommendations: A Query Generation-Based, Training-Free Approach](http://arxiv.org/abs/2504.11889v1), 2025-04-16

(38) [Evaluating the Goal-Directedness of Large Language Models](http://arxiv.org/abs/2504.11844v1), 2025-04-16

(39) [FiSMiness: A Finite State Machine Based Paradigm for Emotional Support Conversations](http://arxiv.org/abs/2504.11837v1), 2025-04-16

(40) [Could Thinking Multilingually Empower LLM Reasoning?](http://arxiv.org/abs/2504.11833v1), 2025-04-16

(41) [Déjà Vu: Multilingual LLM Evaluation through the Lens of Machine Translation Evaluation](http://arxiv.org/abs/2504.11829v1), 2025-04-16

(42) [Cost-Efficient LLM Serving in the Cloud: VM Selection with KV Cache Offloading](http://arxiv.org/abs/2504.11816v1), 2025-04-16

(43) [Efficient and Adaptive Simultaneous Speech Translation with Fully Unidirectional Architecture](http://arxiv.org/abs/2504.11809v1), 2025-04-16

(44) [Résumé abstractif à partir d'une transcription audio](http://arxiv.org/abs/2504.11803v1), 2025-04-16

(45) [Selective Attention Federated Learning: Improving Privacy and Efficiency for Clinical Text Classification](http://arxiv.org/abs/2504.11793v1), 2025-04-16

(46) [Large Language Models for Drug Overdose Prediction from Longitudinal Medical Records](http://arxiv.org/abs/2504.11792v1), 2025-04-16

(47) [Enhancing Web Agents with Explicit Rollback Mechanisms](http://arxiv.org/abs/2504.11788v1), 2025-04-16

(48) [The Digital Cybersecurity Expert: How Far Have We Come?](http://arxiv.org/abs/2504.11783v1), 2025-04-16

(49) [Bridging the Semantic Gaps: Improving Medical VQA Consistency with LLM-Augmented Question Sets](http://arxiv.org/abs/2504.11777v1), 2025-04-16

(50) [Shared Disk KV Cache Management for Efficient Multi-Instance Inference in RAG-Powered LLMs](http://arxiv.org/abs/2504.11765v1), 2025-04-16

(51) [Characterizing and Optimizing LLM Inference Workloads on CPU-GPU Coupled Architectures](http://arxiv.org/abs/2504.11750v1), 2025-04-16

(52) [Climbing the Ladder of Reasoning: What LLMs Can-and Still Can't-Solve after SFT?](http://arxiv.org/abs/2504.11741v1), 2025-04-16

(53) [The Devil is in the Prompts: Retrieval-Augmented Prompt Optimization for Text-to-Video Generation](http://arxiv.org/abs/2504.11739v1), 2025-04-16

(54) [EdgePrompt: A Distributed Key-Value Inference Framework for LLMs in 6G Networks](http://arxiv.org/abs/2504.11729v1), 2025-04-16

(55) [Probing the Unknown: Exploring Student Interactions with Probeable Problems at Scale in Introductory Programming](http://dx.doi.org/10.1145/3724363.3729093), 2025-04-16

(56) [The Hitchhiker's Guide to Program Analysis, Part II: Deep Thoughts by LLMs](http://arxiv.org/abs/2504.11711v1), 2025-04-16

(57) [A Library of LLM Intrinsics for Retrieval-Augmented Generation](http://arxiv.org/abs/2504.11704v1), 2025-04-16

(58) [Progent: Programmable Privilege Control for LLM Agents](http://arxiv.org/abs/2504.11703v1), 2025-04-16

(59) [A New Paradigm of User-Centric Wireless Communication Driven by Large Language Models](http://arxiv.org/abs/2504.11696v1), 2025-04-16

(60) [Can GPT tell us why these images are synthesized? Empowering Multimodal Large Language Models for Forensics](http://arxiv.org/abs/2504.11686v1), 2025-04-16

(61) [Higher-Order Binding of Language Model Virtual Personas: a Study on Approximating Political Partisan Misperceptions](http://arxiv.org/abs/2504.11673v1), 2025-04-16

(62) [Steering Prosocial AI Agents: Computational Basis of LLM's Decision Making in Social Simulation](http://arxiv.org/abs/2504.11671v1), 2025-04-16

(63) [Improving LLM Interpretability and Performance via Guided Embedding Refinement for Sequential Recommendation](http://arxiv.org/abs/2504.11658v1), 2025-04-15

(64) [70% Size, 100% Accuracy: Lossless LLM Compression for Efficient GPU Inference via Dynamic-Length Float](http://arxiv.org/abs/2504.11651v1), 2025-04-15

(65) [Making Acoustic Side-Channel Attacks on Noisy Keyboards Viable with LLM-Assisted Spectrograms' "Typo" Correction](http://arxiv.org/abs/2504.11622v1), 2025-04-15

(66) [GraphicBench: A Planning Benchmark for Graphic Design with Language Agents](http://arxiv.org/abs/2504.11571v1), 2025-04-15

(67) [NodeRAG: Structuring Graph-based RAG with Heterogeneous Nodes](http://arxiv.org/abs/2504.11544v1), 2025-04-15

(68) [REAL: Benchmarking Autonomous Agents on Deterministic Simulations of Real Websites](http://arxiv.org/abs/2504.11543v1), 2025-04-15

(69) [ReTool: Reinforcement Learning for Strategic Tool Use in LLMs](http://arxiv.org/abs/2504.11536v1), 2025-04-15

(70) [HypoBench: Towards Systematic and Principled Benchmarking for Hypothesis Generation](http://arxiv.org/abs/2504.11524v1), 2025-04-15

(71) [DeepMath-103K: A Large-Scale, Challenging, Decontaminated, and Verifiable Mathematical Dataset for Advancing Reasoning](http://arxiv.org/abs/2504.11456v1), 2025-04-15

(72) [TextArena](http://arxiv.org/abs/2504.11442v1), 2025-04-15

(73) [Masculine Defaults via Gendered Discourse in Podcasts and Large Language Models](http://arxiv.org/abs/2504.11431v1), 2025-04-15

(74) [A Dual-Space Framework for General Knowledge Distillation of Large Language Models](http://arxiv.org/abs/2504.11426v1), 2025-04-15

(75) [Reinforcing Compositional Retrieval: Retrieving Step-by-Step for Composing Informative Contexts](http://arxiv.org/abs/2504.11420v1), 2025-04-15

(76) [Efficient Hybrid Language Model Compression through Group-Aware SSM Pruning](http://arxiv.org/abs/2504.11409v1), 2025-04-15

(77) [DataDecide: How to Predict Best Pretraining Data with Small Experiments](http://arxiv.org/abs/2504.11393v1), 2025-04-15

(78) [RankAlign: A Ranking View of the Generator-Validator Gap in Large Language Models](http://arxiv.org/abs/2504.11381v1), 2025-04-15

(79) [Cancer-Myth: Evaluating AI Chatbot on Patient Questions with False Presuppositions](http://arxiv.org/abs/2504.11373v1), 2025-04-15

(80) [OpenTuringBench: An Open-Model-based Benchmark and Framework for Machine-Generated Text Detection and Attribution](http://arxiv.org/abs/2504.11369v1), 2025-04-15

(81) [Teaching Large Language Models to Reason through Learning and Forgetting](http://arxiv.org/abs/2504.11364v1), 2025-04-15

(82) [DataSentinel: A Game-Theoretic Detection of Prompt Injection Attacks](http://arxiv.org/abs/2504.11358v1), 2025-04-15

(83) [Kimina-Prover Preview: Towards Large Formal Reasoning Models with Reinforcement Learning](http://arxiv.org/abs/2504.11354v1), 2025-04-15

(84) [A Minimalist Approach to LLM Reasoning: from Rejection Sampling to Reinforce](http://arxiv.org/abs/2504.11343v1), 2025-04-15

(85) [Optimizing LLM Inference: Fluid-Guided Online Scheduling with Memory Constraints](http://arxiv.org/abs/2504.11320v1), 2025-04-15

(86) [Learning to Be A Doctor: Searching for Effective Medical Agent Architectures](http://arxiv.org/abs/2504.11301v1), 2025-04-15

(87) [Automated Python Translation](http://arxiv.org/abs/2504.11290v2), 2025-04-15

(88) [The Obvious Invisible Threat: LLM-Powered GUI Agents' Vulnerability to Fine-Print Injections](http://arxiv.org/abs/2504.11281v1), 2025-04-15

(89) [From Misleading Queries to Accurate Answers: A Three-Stage Fine-Tuning Method for LLMs](http://arxiv.org/abs/2504.11277v1), 2025-04-15

(90) [Distillation-Supervised Convolutional Low-Rank Adaptation for Efficient Image Super-Resolution](http://arxiv.org/abs/2504.11271v1), 2025-04-15

(91) [Towards Automated Safety Requirements Derivation Using Agent-based RAG](http://arxiv.org/abs/2504.11243v1), 2025-04-15

(92) [Nondeterministic Polynomial-time Problem Challenge: An Ever-Scaling Reasoning Benchmark for LLMs](http://arxiv.org/abs/2504.11239v1), 2025-04-15

(93) [AutoRAN: Automated and Zero-Touch Open RAN Systems](http://arxiv.org/abs/2504.11233v1), 2025-04-15

(94) [Video Summarization with Large Language Models](http://arxiv.org/abs/2504.11199v1), 2025-04-15

(95) [Enhancing Autonomous Driving Systems with On-Board Deployed Large Language Models](http://arxiv.org/abs/2504.11514v1), 2025-04-15

(96) [Enhancing multimodal analogical reasoning with Logic Augmented Generation](http://arxiv.org/abs/2504.11190v1), 2025-04-15

(97) [Benchmarking Next-Generation Reasoning-Focused Large Language Models in Ophthalmology: A Head-to-Head Evaluation on 5,888 Items](http://arxiv.org/abs/2504.11186v1), 2025-04-15

(98) [Exploring Backdoor Attack and Defense for LLM-empowered Recommendations](http://arxiv.org/abs/2504.11182v1), 2025-04-15

(99) [MuSeD: A Multimodal Spanish Dataset for Sexism Detection in Social Media Videos](http://arxiv.org/abs/2504.11169v1), 2025-04-15

(100) [Bypassing Prompt Injection and Jailbreak Detection in LLM Guardrails](http://arxiv.org/abs/2504.11168v2), 2025-04-15
